squarelet/users/apps.py

```diff
@@ -31,3 +31,8 @@ def ready(self):
         except ProgrammingError:
             # skip if RSA Key is not found for some reason
             pass
+
+        from allauth.account import signals as account_signals
+        from allauth.mfa import signals as mfa_signals
+
+        account_signals._add_email.disconnect(mfa_signals.on_add_email)
```

Comment:
Would be good to leave a comment here for why we're disconnecting the signal handler.